web/src/hitsByIncomingLink.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { List, Datagrid, FunctionField, NumberField } from "react-admin";
+
+if (!process.env.ABSOLUTE_LINK_PREFIX) {
+  throw new Error("ABSOLUTE_LINK_PREFIX is not set");
+}
+
+const absoluteLinkPrefix = process.env.ABSOLUTE_LINK_PREFIX;
```

Comment:
Instead of duplicating, I suggest moving this + `toAbsoluteIncomingLink` to a file that the app can use from anywhere.